### ISSUE TITLE
Feature Request: Add option to newline-per-chained-call for func calls only

### ISSUE THREAD

**Issue by thedark1337** [eslint/eslint#5289]
Personally , I want to ask if its possible to have an option to chain function calls only and not object access as well?

Example:

```
// valid with option on, invalid with off
db.user
            .query()
            .then()
            .catch();
```

I'm not sure what to name the option, perhaps `funcCallOnly` ? e.g 

```
  'newline-per-chained-call': [2, {funcCallOnly: true}]
```

**Comment by BYK** [comment]
I'd prefer `onlyMethods` or `memberAccess` or `propertyAccess`. That said the initial explanation of the rule just mentions method calls so may be we should exclude member access completely. That would be a breaking change though.

**Comment by nzakas** [comment]
I'm pretty sure this is a bug, the rule even says "chainable call".

**Comment by mtpc** [comment]
Bumping this, since I often face this issue myself.

**Comment by khakulov** [comment]
This code causes error by me
```javascript
// error, but no chained function call, wtf?
const debug = window.location.search.indexOf('debug=1') !== -1;

// no error
const debug = window
    .location
    .search
    .indexOf('debug=1') !== -1;
```
i think also, this is a bug

**Comment by BYK** [comment]
Working on this.

**Comment by briandipalma** [comment]
FWIW I also think this is a bug, I see @BYK fixed the issue and it's just waiting for an OK from @gajus and @rpatil26 to be merged in? Does this mean we should just disable the rule or ignore the warnings until the next release of ESLint?
